joss/paper.md

```diff
@@ -0,0 +1,163 @@
+from the surface of a neutron star. Pulsing X-ray signals from neutron stars
+are modeled to statistically estimate parameters such as stellar mass and
+radius, and properties of the surface radiation field such as a map of
+temperature. The mass and radius of a neutron star are a function of the
+equation of state of internal matter (especially the dense matter in the core)
+and the formation history of the star (that determines the central energy
+density and the spin frequency); the state of the surface radiation field is
+the product of a potentially long and complex stellar evolutionary history,
+especially that of the stellar magnetosphere. Such parameter estimation
+requires relativistic tracing of radiation as it propagates from surface to a
+distant telescope.
+
+There are a small number of open-source libraries for simulating the X-ray
+signals from rapidly spinning neutron stars and more generally from the
+vicinity of general relativistic compact objects (including black holes), but
+the scope of these projects does not include statistical modeling, which
+necessitates tractable parametrised models and a modular framework for
+constructing those models. The state-of-the-art example is the Arcmancer
+library of @Pihajoki:2018, a general purpose toolbox that is wrapped by an
+updated version of the bender library of @Nattila:2016 for the purpose of
+simulating X-ray signals from hot regions on the surfaces of rapidly rotating
+neutron stars. These general purpose libraries have a wide array of
+applications and would exhibit synergy with a library for statistical modeling
+of X-ray signals registered by space telescopes. The general purpose libraries
+are for example useful for accurate simulations of realistic, general
+relativistic neutron stars. A statistical modeling library would be useful for
+distilling such state-of-the-art models down to the principal components and
+forming much more approximative models, with a small number of parameters, that
+are nevertheless sufficient to generate data that is structured like the
+observational data; using such models, one can derive probabilistic information
+that represents our knowledge of neutron star properties.
+
+Likelihood information about the mass and radius of a given neutron star is
+utilized in an array of contexts. A pertinent example is post-processing of the
+likelihood into information about the equation of state of cold dense matter
+inside neutron stars. Likelihood information about the properties of a surface
+radiation field can be post-processed, for instance, into information about a
+stellar magnetosphere. Contemporary and future telescopes are taking and would
+take aim at these pulsing stars in order to derive such information: the most notable
+example is the NICER (Neutron star Interior Composition ExploreR) X-ray
+telescope, a NASA payload installed on the International Space Station [@Gendreau2016].
```

Comment:
This paragraph seems very convoluted? I would probably not start with likelihood information here, because we've not really talked about likelihoods earlier. Maybe just "probabilistic modelling is used to infer information about the equation of state of cold dense matter inside neutron stars and to help us understand the stellar magnetosphere"?

Reply:
Text updated.